### PR TITLE
[NR-76990] V6.9.2 release notes, EOL and DG docs update

### PR DESCRIPTION
**Pull request by ywang-nr**
## Give us some context

* What problems does this PR solve? - Android Agent v6.9.2 doc updates

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

**Comment by homelessbirds** [comment]
@ywang-nr let us know when you're ready for us to review this!

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-692.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+releaseDate: '2023-01-11'
+version: 6.9.2
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.9.2.zip'
+---
+
+## Fixed
+
+* Fixed an issue of missing element in harvest payload, which has caused data loss during ingest in version [6.9.1](/docs/release-notes/mobile-release-notes/android-release-notes/android-691)
+* Tightened metric data validations to prevent data loss during ingest
+* Fixed a bug with the plugin fails to tag and upload 'mapping.txt' file
```
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-692.mdx:12]: This statement doesn't accurately describe the problem, but was more a symptom. I'd like to change it to:

"Fixed a bug that allowed direct access to the agent plugin extension's configuration from Kotlin that could result in a failure to tag and upload obfuscation maps."

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Comment by ywang-nr** [comment]
1/9/2023 Status: Merged Kenny's changes and updated wording

**Review by cthomas-newrelic — APPROVED** [review]
Looks great!

**Comment by ywang-nr** [comment]
> @ywang-nr let us know when you're ready for us to review this!

@homelessbirds this PR is ready to review by the doc team, target to release this afternoon(1/10/2023)

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx [hunk 2]
```diff
@@ -134,7 +134,8 @@ To add support for **DexGuard**:
    ```
    buildTypes {
            release {
-              minifyEnabled true
+              minifyEnabled false
+              shrinkResources false
```
Comment (ywang-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx:138]: do we need it or not?

**Review by bradleycamacho — APPROVED** [review]

**Pull request merged**